spec/forms/openid_connect_token_form_spec.rb

```diff
@@ -409,7 +409,11 @@
 
     context 'with valid params' do
       before do
-        OutOfBandSessionAccessor.new(identity.rails_session_id).put_pii({}, 5.minutes.to_i)
+        OutOfBandSessionAccessor.new(
+          identity.rails_session_id,
+        ).put_empty_user_session(
+          5.minutes.to_i,
```

Comment:
Noticed that this argument has `to_i`, but the default argument above is just `5.minutes`, not sure if that's an issue.

Reply:
I don't think it is. I'm just trying to keep it consistent with what was there before.